docs/troubleshooting/harvester.md

```diff
@@ -67,6 +67,121 @@ Users can generate a support bundle in the Harvester GUI with the following step
 - Enter a useful description for the support bundle and click `Create` to generate and download a support bundle.
     ![](/img/v1.2/troubleshooting/harvester-sb-support-modal.png)
 
+### Retain Support Bundle File and Download Manually
+
+By default the support bundle file is generated, downloaded and deleted automatically after users click the `Create` button.
+
+There are also scenarios that users want to keep the support bundle files for some time or re-download the interrupted downloading.
+
+- Users fail to download the support bundle file due to the slow network performance, network disruption or other reasons, they want to re-download the support-file. Even though the support bundle file remains in the cluster, the Harvester WebUI does not have a place to download the file.
+
+- It takes a long time to generate the support bundle file, users want to reuse a previous support bundle file for quick troubleshotting.
+
+
+The workaround is:
+
+1. Break the automatic downloading.
+
+- Click `Generate Support Bundle` from Harvester WebUI.
+
+- When the progress is about 20%, close the web page.
+
+- Check support bundle status from kubectl.
+
+```
+# kubectl get supportbundle -A -oyaml
```

Comment:
```suggestion
1. Generate the file and prevent automatic downloading.

  1. On the Harvester UI, click **Generate Support Bundle**.

  1. When the progress indicator reaches 20%, close the browser tab to prevent automatic downloading of the generated file.

  1. Retrieve a list of all support bundles in all namespaces using kubectl.

  # kubectl get supportbundle -A
  NAMESPACE          NAME           ISSUE_URL   DESCRIPTION   AGE
  harvester-system   bundle-htl5f               sp1           3h43m

  1. Identify the resource name of the file that you want to download and then check its status.
```

Reply:
Please add backticks to the code block.